Update README.md (#63)
Qiskit/textbook: README.md

# Qiskit Textbook content

> > :warning: Unfortunately we're not accepting new pages to the textbook. If you

Author: ```suggestion
> :warning: Unfortunately **we can't accept new pages to the textbook**. If you
```
